src/Timer.cpp

```diff
@@ -62,11 +62,19 @@ bool Timer::isStarted() const {
 }
 
 bool Timer::operator<(const Timer &other) const {
-	return uiStart > other.uiStart;
+	return uiStart < other.uiStart;
```

Comment:
I don't think this fix is correct. Previously a smaller elapsed time would compare ```<``` as documented for the function, now it wouldn't.